src/piece/Pawn.hpp

```diff
@@ -8,7 +8,8 @@ namespace chesspp
 {
     namespace piece
     {
-        class Pawn : public virtual Piece
+        class Pawn
```

Comment:
What was your rationale for this change? I don't think we ever inherit from more than one class, so the line shouldn't be getting long and this goes against every code base I've seen. Not that I'm totally against it, I just don't think it makes anything cleaner or messier.